Doc/library/types.rst

```diff
@@ -82,6 +82,32 @@ Dynamic Type Creation
 
    .. versionadded:: 3.7
 
+.. function:: get_original_bases(cls, /)
+
+    Return the objects in the bases list of the class's definition as they
+    existed prior to any modification by :meth:`~object.__mro_entries__`. This
+    is useful for introspecting :ref:`Generics <user-defined-generics>`.
```

Comment:
How about this, to address @gvanrossum's feedback?

```suggestion
    Return the tuple of objects originally given as the bases of *cls* before
    the :meth:`~object.__mro_entries__` method has been called on any bases,
    following the mechanisms laid out in :pep:`560`. This is useful for
    introspecting :ref:`Generics <user-defined-generics>`.

    For classes that have an :attr:`~type.__orig_bases__` attribute, this
    function simply returns the value of ``cls.__orig_bases__``.
```

Reply:
Isn't 
>     For classes that have an :attr:`~type.__orig_bases__` attribute, this
going to not resolve properly?

Reply:
> Isn't
> 
> > ```
> > For classes that have an :attr:`~type.__orig_bases__` attribute, this
> > ```
> 
> going to not resolve properly?

I changed my review suggestion to not include that :)